CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ else()
     include(GetGitRevisionDescription)
     git_describe(DETHRACE_VERSION)
 endif()
-message(STATUS "DethRace version ${DETHRACE_VERSION}")
+
+message(STATUS "dethrace version ${DETHRACE_VERSION}")
```

Comment:
Just a reminder if you ever decide to create a new dethrace release, to not forget to include a `VERSION` file in the source release.